src/adapter/z-stack/adapter/zStackAdapter.ts

```diff
@@ -918,6 +918,27 @@ class ZStackAdapter extends Adapter {
         });
     }
 
+    public async supportsSwitchChannel(): Promise<boolean> {
+        return false;
+    }
+
+    public async switchChannel(newChannel: number): Promise<void> {
+        return this.queue.execute<void>(async () => {
+            this.checkInterpanLock();
+
+            const payload = {
+                dstaddr: 0xFFFF,// broadcast with sleepy
+                dstaddrmode: AddressMode.ADDR_BROADCAST,
+                channelmask: [newChannel].reduce((a, c) => a + (1 << c), 0),
+                scanduration: 0xFE,/*switch channel*/
+                // scancount: null,// TODO: what's "not present" here?
```

Comment:
Note to self; we can use the values from https://github.com/zigpy/zigpy-znp/blob/e3c9fcfdd249e089c578f99c4ed22159e9325c42/zigpy_znp/zigbee/application.py#L369

Reply:
See 2.4.3.3.9.2 of [spec](https://csa-iot.org/wp-content/uploads/2023/04/05-3474-23-csg-zigbee-specification-compressed.pdf) for the flow on reception.